docs/content/latest/explore/security/security.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Security
+headerTitle: Security
+linkTitle: Security
+description: Overview of security in YugabyteDB.
+headcontent: Security in YugabyteDB.
+image: /images/section_icons/index/secure.png
+aliases:
+  - /latest/explore/security/
+menu:
+  latest:
+    identifier: explore-security
+    parent: explore
+    weight: 930
+isTocNested: true
+showAsideToc: true
+---
+
+Like PostgreSQL, YugabyteDB provides security in multiple ways:
+
+* **Authentication** - limit access to the database to clients with proper credentials
+
+* **Authorization** - establish roles and privileges to restrict activities that clients can perform
+
+* **Encryption** - encrypt the database, as well as all network communication between servers
+
+* **Auditing** - conduct session- and object-level auditing
+
+* **Restricting network access** - limit connections to the database using RPC binding
+
+## Authentication
+
+Authentication is the process by which the database server establishes the identity of the client, and by extension determines whether the client application (or the user who runs the client application) is permitted to connect with the database user name that was requested. YugabyteDB offers a number of different client authentication methods, including:
```

Comment:
Suggestion:
...  including the following:

Reply:
Fixed